share/functions/funced.fish

```diff
@@ -1,3 +1,17 @@
+function __funced_md5
+    if type -q md5sum
+        # GNU systems
+        md5sum $argv[1] | cut -d' ' -f1
+        return 0
+    end
+    if type -q md5
```

Comment:
This should be an "else if", otherwise if both exist, both will be run, which is at the very least a waste of resources.